holidays/locale/en_US/LC_MESSAGES/MU.po

```diff
@@ -0,0 +1,100 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see CONTRIBUTORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+#
+# Mauritius holidays en_US localization.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Holidays 0.76\n"
+"POT-Creation-Date: 2025-06-15 18:34+0530\n"
+"PO-Revision-Date: 2025-06-15 18:34+0530\n"
+"Last-Translator: Kriti Birda <workwithkritibirda@gmail.com>\n"
+"Language-Team: Holidays Localization Team\n"
+"Language: en_US\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Generated-By: Lingva 5.0.6\n"
+"X-Source-Language: en_MU\n"
+
+#. %s (estimated).
+#, c-format
+msgid "%s (estimated)"
+msgstr "%s (estimated)"
+
+#. New Year's Day.
+msgid "New Year's Day"
+msgstr "New Year's Day"
+
+#. Day after New Year's Day.
+msgid "Day after New Year's Day"
+msgstr "Day after New Year's Day"
+
+#. Abolition of Slavery.
+msgid "Abolition of Slavery"
+msgstr "Abolition of Slavery"
+
+#. Independence and Republic Day.
+msgid "Independence and Republic Day"
+msgstr "Independence and Republic Day"
+
+#. Labor Day.
+msgid "Labour Day"
+msgstr "Labor Day"
+
+#. Arrival of Indentured Labourers.
+msgid "Arrival of Indentured Labourers"
+msgstr "Arrival of Indentured Labourers"
+
+#. All Saints' Day.
+msgid "All Saints' Day"
+msgstr "All Saints' Day"
+
+#. Assumption of the Blessed Virgin Mary.
+msgid "Assumption of the Blessed Virgin Mary"
+msgstr "Assumption of the Blessed Virgin Mary"
```

Comment:
Current `en_US` usage is mostly either "Assumption of Mary Day" or just "Assumption Day" AFAIK